stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -488,28 +486,48 @@ public struct ${Self}<Pointee>
   }
 
 %  if mutable:
+  @available(swift, obsoleted: 5.0.0, renamed: "initialize(repeating:count:)")
+  public func initialize(to repeatedValue: Pointee, count:Int = 1) { 
```

Comment:
Nit: space after `count:`.

Also, the function that's being obsoleted is technically `initialize(to newValue: Pointee, count: Int = 1)` (as opposed to `repeatedValue`). Doesn't strictly matter for compatibility, but the name is more correct in the circumstance that `count` defaults to 1.

Reply:
done